setup.py

```diff
@@ -44,12 +44,8 @@ def find_version(*parts):
           ],
```

Comment:
in `package_data` should we also include the dssc geometry file?

Reply:
Yes, good catch, thanks.